doc/reference/tooling/tt_cli/download.rst

```diff
@@ -0,0 +1,65 @@
+.. _tt-download:
+
+Downloading Tarantool Enterprise SDK
+====================================
+
+..  code-block:: console
+
+    $ tt download VERSION [OPTION ...]
+
+``tt download`` downloads Tarantool Enterprise SDK from the customer zone.
+
+The ``VERSION`` is a part of the SDK archive name between ``tarantool-enterprise-sdk-``
+and the platform identifier. For example, to download ``tarantool-enterprise-sdk-gc64-3.0.0-0-gf58f7d82a-r23.linux.x86_64.tar.gz``,
+run:
+
+..  code-block:: console
+
+    $ tt download gc64-3.0.0-0-gf58f7d82a-r23
+
+``tt`` automatically chooses the bundle for the current platform.
+
+.. _tt-download-authentication:
+
+Authentication
+~~~~~~~~~~~~~~
+
+To download the Tarantool Enterprise SDK using ``tt download``, you need to provide
+access credentials for Tarantool customer zone. Use one of the following ways to pass
+the username and the password:
+
+*   A text file specified in the ``ee.credentials_path`` parameter of the
```

Comment:
`credentials_path` -> `credential_path` ? (without S)
The same for `tt install`.